src/moby/build.go

```diff
@@ -145,6 +145,11 @@ func Build(m Moby, w io.Writer, pull bool, tp string) error {
 		MobyDir = defaultMobyConfigDir()
 	}
 
+	// create tmp dir in case needed
+	if err := os.MkdirAll(filepath.Join(MobyDir, "tmp"), 0755); err != nil {
```

Comment:
`/tmp` is usually mode `1777`, isn't it?

Reply:
this is not `/tmp` it is under your home dir

Reply:
Specifically it is (by default) `~/.moby/tmp`. OK then.